starknet_py/tests/e2e/tests_on_networks/client_test.py

```diff
@@ -65,11 +54,6 @@ async def test_wait_for_tx_reverted_full_node(full_node_account_integration):
         await account.client.wait_for_tx(tx_hash=invoke.transaction_hash)
 
 
-# Same here as in comment above 'test_wait_for_tx_reverted_full_node'
-@pytest.mark.skipif(
-    condition="--client=gateway" in sys.argv,
-    reason="Gateway client has been disabled on integration network.",
-)
 @pytest.mark.asyncio
 async def test_wait_for_tx_full_node_accepted(full_node_account_integration):
```

Comment:
In order to stay consistent (my bad here 😅), renaming to suggestion below (or renaming the test above to match   name of this test) would be great:

```suggestion
async def test_wait_for_tx_accepted_full_node(full_node_account_integration):
```

Reply:
I guess we can also remove `_full_node` from the function name